README.md

```diff
@@ -1,76 +1,6 @@
 # .NET Versions
-This repo contains information about the various component versions that ship with .NET Core. 
 
-## build-info
```

Comment:
The new info about the 3 build-info/ subfolders is great and I agree `Maestro` had to go.

Suggest restoring this information or moving it into a new build-info/ readme. Losing about the only way to navigate that tree other than blindly searching around sub-folders. Admitedly some of the terminology is Maestro-centric but most of the names appear as file and folder names w/in build-info/.

Reply:
I'll move the info into dotnet and label it as legacy information. I was worried that the existence of it implies that any of it is still active. We haven't done a build that uses this infra in many many years.